projects/plugins/jetpack/modules/search/instant-search/loader.js

```diff
@@ -0,0 +1,28 @@
+// NOTE: This must be imported first before any other imports.
+// See: https://github.com/webpack/webpack/issues/2776#issuecomment-233208623
+import './set-webpack-public-path';
+
+/**
+ * Internal dependencies
+ */
+import { SERVER_OBJECT_NAME } from './lib/constants';
+import { bindCustomizerChanges } from './lib/customize';
+
+/**
+ * Loads and runs the main chunk for Instant Search.
+ */
+function init() {
+	import( './index' ).then( instantSearch => instantSearch.initialize() );
+}
+
+// Bind customizer changes immediately.
+if ( window[ SERVER_OBJECT_NAME ] ) {
+	bindCustomizerChanges();
```

Comment:
Note that I'm assuming that this code needs to execute immediately before DCL, since that's what was being done before. If that's not the case, the size of the initial chunk can be improved further by moving this inside `initialize`.